src/Raven.Server/Documents/ETL/EtlProcess.cs

```diff
@@ -1328,6 +1329,21 @@ public override OngoingTaskConnectionStatus GetConnectionStatus()
                                         result = rabbitMqEtl.RunTest(results, context);
                                         result.DebugOutput = debugOutput;
 
+                                        return tx;
+                                    }
+                                case AzureQueueStorageEtl azureQueueStorageEtl:
+                                    using (azureQueueStorageEtl.EnterTestMode(out debugOutput))
+                                    {
+                                        azureQueueStorageEtl.EnsureThreadAllocationStats();
+
+                                        var queueItem = new QueueItem(document, docCollection);
+
+                                        var results = azureQueueStorageEtl.Transform(new[] { queueItem }, context, new EtlStatsScope(new EtlRunStats()),
+                                            new EtlProcessState());
+
+                                        result = azureQueueStorageEtl.RunTest(results, context);
+                                        result.DebugOutput = debugOutput;
```

Comment:
For all of the queue ETLs the code is the same. Can we avoid duplicating it?

Reply:
There is open PR which touches this area as well - https://github.com/ravendb/ravendb/pull/18375